### PR TITLE
Rename golang module

### PR DESCRIPTION
**Pull request by sabre1041**
Rename golang module to align with repository name change

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,45 +5,45 @@ GO_BUILD_BINDIR :=./bin
...
-				   -X github.com/uor-framework/client/cli.commit="$(GIT_COMMIT)" \
-				   -X github.com/uor-framework/client/cli.buildDate="$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')"
+GO_LD_EXTRAFLAGS :=-X github.com/uor-framework/uor-client-go/cli.version="$(shell git tag | sort -V | tail -1)" \
+				   -X github.com/uor-framework/uor-client-go/cli.buildData="dev" \
+				   -X github.com/uor-framework/uor-client-go/cli.commit="$(GIT_COMMIT)" \
+				   -X github.com/uor-framework/uor-client-go/cli.buildDate="$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')"
 
 build: prep-build-dir
-	$(GO) build -o $(GO_BUILD_BINDIR)/client  -ldflags="$(GO_LD_EXTRAFLAGS)" $(GO_BUILD_PACKAGES)
+	$(GO) build -o $(GO_BUILD_BINDIR)/uor-client-go  -ldflags="$(GO_LD_EXTRAFLAGS)" $(GO_BUILD_PACKAGES)
```
Comment (jpower432) [inline Makefile:14]: Do we want to keep the binary name `client` for now?

**Review by jpower432 — COMMENTED** [review]
Everything looks good. One question, this changes the binary name to `uor-client-go` along with all of the examples and documentation. Should we switch that to `uor-client` or leave it as `client` for now?

File: Makefile (hunk 1 above)
Comment (sabre1041) [reply Makefile:14]: Client is so generic. So we would want to align to just the repo name or a more long term name of the client

Comment (jpower432) [reply Makefile:14]: I am good with just using the repo name for now until we determine the long-term name.

Comment (sabre1041) [reply Makefile:14]: Sounds good. As soon as we want to release, we should update website content (I can do that)

**Review by jpower432 — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (usrbinkat) [reply Makefile:14]: I'm interested in this conversation.

At different points, we've discussed this repo serving as a reference implementation and not a formal "product" producing repository. We have also discussed taking on a [kaiju](https://en.wikipedia.org/wiki/Kaiju) based branding theme and the client becoming something like `kaictl`.

Should we go ahead and document some formal outcomes to make a decision on this?

**Pull request merged**